### PR TITLE
[Messenger][AmazonSQS] Add `AddFifoStamp` middleware

### LINKED ISSUE #19050: [Messenger] [Sqs] Add `AddFifoStamp` middleware
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#48095
| PR author(s) | @tyx
| Merged in    | 6.4

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.4 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/19050

### CODE REVIEW

File: messenger.rst [hunk 1]
```diff
@@ -1810,6 +1810,17 @@ The transport has a number of options:
     If the queue name is suffixed by ``.fifo``, AWS will create a `FIFO queue`_.
     Use the stamp :class:`Symfony\\Component\\Messenger\\Bridge\\AmazonSqs\\Transport\\AmazonSqsFifoStamp`
     to define the ``Message group ID`` and the ``Message deduplication ID``.
+    You can also enable the
```
Comment (OskarStark) [inline messenger.rst]: 1. This is only useful when using AWS
2. We should explain more what it does internally, when using it

Comment (alexandre-daubois) [reply messenger.rst]: This bit has been added to the SQS part, so I guess your first point is resolved :slightly_smiling_face: 

I also reworded to be more precise on what this middleware does internally. I missed the two new interfaces. Thanks!

**Comment by OskarStark** [comment]
Thank you Alexandre.

**Comment by tyx** [comment]
Thank you !

**Pull request merged**